config/ie/options

```diff
@@ -1,382 +1,391 @@
-##########################################################################
```

Comment:
Something weird is going on here - GitHub is showing a diff for every single line, as is git locally. Are the line endings maybe messed up or something?

Reply:
Not sure what happened here but I've reapplied the changes in https://github.com/GafferHQ/gaffer/pull/2328/commits/e3e40d4b7bc554618fb15bcf8d0030059d5d091f and it hasn't replaced the line endings.